### PR TITLE
Set cypress downloads and videos folders.

### PR DESCRIPTION
**Pull request by mshima**
Fixes https://github.com/jhipster/generator-jhipster/issues/18536.

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/cypress/templates/cypress.json.ejs [hunk 1]
```diff
@@ -25,6 +25,8 @@
   "fixturesFolder": "<%= CLIENT_TEST_SRC_DIR %>cypress/fixtures",
   "pluginsFile": "<%= CLIENT_TEST_SRC_DIR %>cypress/plugins/index.ts",
   "screenshotsFolder": "<%= BUILD_DIR %>cypress/screenshots",
+  "downloadsFolder": "<%= BUILD_DIR %>cypress/downloads",
+  "videosFolder": "<%= BUILD_DIR %>cypress/screenshots",
```
Comment (mraible) [inline generators/cypress/templates/cypress.json.ejs:29]: Should this be videos instead of screenshots?

Comment (mshima) [reply generators/cypress/templates/cypress.json.ejs:29]: Sure
```suggestion
  "videosFolder": "<%= BUILD_DIR %>cypress/videos",
```

**Comment by mraible** [comment]
It's OK to specify then. However, I'm more interested in ignoring them. Downloads, screenshots, and videos don't seem like something you'd want in source control.

**Comment by mshima** [comment]
`BUILD_DIR` is `target` for maven and `build` for gradle.
They are ignored by git.
We already put node, webpack cache and others temporary directories there.

**Review by mraible — APPROVED** [review]

**Pull request merged**